packages/grafana-ui/src/components/Table/FooterRow.tsx

```diff
@@ -28,32 +30,25 @@ export const FooterRow = (props: FooterRowProps) => {
     >
       {footerGroups.map((footerGroup: HeaderGroup) => {
         const { key, ...footerGroupProps } = footerGroup.getFooterGroupProps();
-        let style = {};
-        if (footers && footers.length > 1) {
-          const height = EXTENDED_ROW_HEIGHT * footers.length;
-          style = { height: `${height}px` };
-        }
-        {
-          return (
-            <div
-              className={tableStyles.tfoot}
-              {...footerGroupProps}
-              key={key}
-              aria-label={e2eSelectorsTable.footer}
-              style={style}
-            >
-              {footerGroup.headers.map((column: ColumnInstance, index: number) =>
-                renderFooterCell(column, tableStyles, style)
-              )}
-            </div>
-          );
-        }
+        return (
+          <div
```

Comment:
We have a strong focus on a11y now and I'm not sure a `div` is the correct HTML semantics for a footer. Can you investigate what would be the best alternative here?

Reply:
Also updated the header which was div based.  Just added the proper roles there.